README.md

```diff
@@ -100,11 +60,7 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **A database that allows for querying and filtering based on it's features.**
 
```

Comment:
- [ ] At the moment, your Live demo link is broken. You can write the coming soon text simply without making it a link. Kindly fix this